tests/test_transforms.py

```diff
@@ -13,6 +13,7 @@ def test_univariate_transforms():
         IdentityTransform(),
         CosTransform(),
         SinTransform(),
+        ComposeTransform([IdentityTransform()]),
```

Comment:
As `ComposeTransform` is not implemented by Zuko, I don't think it should be tested here.

Reply:
```suggestion
```

Reply:
Well, that was the regression test to amke sure that it does not occur again. And it is changed by the library (due to monkey-patching `Transform.call_and_ladj`). That's why I added it here - to make sure the error is not re-introduced.